Documentation/typo3fluid/fluid/latest/Section.rst

```diff
@@ -61,6 +61,14 @@ Special sections HeaderAssets/FooterAssets
 In case you want to add header or footer assets within plugins, you can use the special sections "HeaderAssets" and "FooterAssets".
 Those are rendered by the ActionController (\TYPO3\CMS\Extbase\Mvc\Controller\ActionController) and provide the possibilty to add resources, meta data etc. as you like by simply using those two sections as wanted.
 
+::
+
+    <f:section name="HeaderAssets">
+       <title>My title within the plugin template</title>
```

Comment:
<link rel="stylesheet" href="typo3conf/ext/my_extension/Resources/Public/Css/myCssFile.css" />

Reply:
the title-tag might be shown twice (with different content), if not the whole header-area is rendered individually.